acceptance/tests/runs_puppet_facts.rb

```diff
@@ -0,0 +1,30 @@
+test_name "facter -p loads facts from puppet"
```

Comment:
Is there a good way to guard on finding puppet? I can envision test environments where we only build Facter to run acceptance.